edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-control-numbers.adoc

```diff
@@ -0,0 +1,28 @@
+= EDIFACT EDI Connector 2.2 - Control Numbers - Mule 4
+
+Using the Write operation in the Anypoint EDIFACT EDI Connector, you can generate control numbers on the generated EDIFACT payload by:
+
+* Managing global control numbers through control number keys
+* Managing identifier combination control numbers
+* Using your own control number
+
+== Manage Global Control Numbers Through Control Number Keys
+
+If your organization or your trading partner have must maintain the uniqueness of interchange, group, or transaction set control numbers across different identifier combinations and document types, then:
+
+* Set the configuration `Use supplied values` to `false`.
+* Leverage the control number key configurations (at the configuration level or operation level) to set a key unique to the partner. +
+By default, the control number key configurations pass dynamically at the operation level. You can then use this group control number key across different identifiers and transaction types for the same partner.
```

Comment:
This is longish for a bullet list item. Consider whether it is better to leave it as is or to insert a line so that the list is more scannable:
```suggestion
* Leverage the control number key configurations (at the configuration level or operation level) to set a key unique to the partner. +
+
By default, the control number key configurations pass dynamically at the operation level. You can then use this group control number key across different identifiers and transaction types for the same partner.
```